client/components/section-nav/index.jsx

```diff
@@ -61,7 +62,7 @@ var SectionNav = React.createClass( {
 			className = classNames( {
 				'section-nav': true,
 				'is-empty': true
-			} );
+			}, this.props.className );
```

Comment:
Just to clarify -- do you mean why we need to pass `className` prop to `SectionNav` at all or why we need to use it in the case if `SectionNav` doesn't have any `children`?